src/app/upload/batch-upload/batch-upload.component.ts

```diff
@@ -0,0 +1,64 @@
+import {Component, HostListener, OnInit} from '@angular/core';
+import {FileSystemDirectoryEntry, FileSystemFileEntry, UploadEvent, UploadFile} from 'ngx-file-drop';
+import {UploadService} from '../upload.service';
+import {PendingScanId} from '../../entities/pending-scan';
+
+@Component({
+  selector: 'app-batch-upload',
+  templateUrl: './batch-upload.component.html',
+  styleUrls: ['./batch-upload.component.scss']
+})
+export class BatchUploadComponent implements OnInit {
+
+  uploadActive = false;
+  private files: File[];
```

Comment:
If we started the underscore prefix for private members, let's stick to it.

Reply:
Thanks to your comment I found a bug. It should be public since it is accessed from the template :)